source/app/powertabeditor.cpp

```diff
@@ -2171,21 +2171,28 @@ void PowerTabEditor::createCommands()
     // Note-related actions.
     myNoteDurationGroup = new QActionGroup(this);
     createNoteDurationCommand(myWholeNoteCommand, tr("Whole"), "Notes.WholeNote",
-                              Position::WholeNote);
+                              Position::WholeNote,
+                              ":images/whole_note");
```

Comment:
These changes look good!  It would be great to also use a [QStringLiteral](https://doc.qt.io/qt-5/qstring.html#QStringLiteral) for these strings since they're constants

Reply:
I did this wherever the icon strings show up.